### PR TITLE
[UI] Invite user to login in /profile when logged out #1581

### PR DESCRIPTION
**Pull request by km-js**
I added the message which the user can see while logged out. Please let me know if we need any other additions to this.

### CODE REVIEW

File: ui/src/views/Profile/index.jsx [hunk 1]
```diff
@@ -37,6 +37,9 @@ export default class Profile extends Component {
       <Dashboard title="Profile">
         {!currentScopes && loading && <Spinner loading />}
         <ErrorPanel fixed error={error} />
+        <Typography variant="subtitle1">
+          Sign in to view your Profile
```
Comment (helfi92) [inline ui/src/views/Profile/index.jsx]: "Profile" can be lower case.

File: ui/src/views/Profile/index.jsx [hunk 2]
```diff
@@ -37,6 +37,11 @@ export default class Profile extends Component {
       <Dashboard title="Profile">
         {!currentScopes && loading && <Spinner loading />}
         <ErrorPanel fixed error={error} />
+        {!user && (
```
Comment (helfi92) [inline ui/src/views/Profile/index.jsx]: `{!user && !loading && (`

File: ui/src/views/Profile/index.jsx (hunk 1 above)
Comment (helfi92) [reply ui/src/views/Profile/index.jsx]: This is still not done. `profile` instead of `Profile`.

Comment (km-js) [reply ui/src/views/Profile/index.jsx]: Sorry for that, it's done now!

**Pull request merged**